vnext/templates/cpp-app/windows/MyApp/resource.h

```diff
@@ -0,0 +1,18 @@
+//{{NO_DEPENDENCIES}}
+// Microsoft Visual C++ generated include file.
+// Used by {{ name }}.rc
+
+#define IDC_RNTESTER_COMPOSITION 100
```

Comment:
RNTESTER reference?

Reply:
There's lots of potential cleanup available that may be subsumed by larger refactors anyway, but tracking bug here: https://github.com/microsoft/react-native-windows/issues/12179